src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
[enum naming should be in singular](https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes):